### DISCUSSION TITLE
Version 0.8.7

### DISCUSSION THREAD

**Discussion by guydavis** [guydavis/machinaris#858] [Announcements]
Hi all! The newest version of Machinaris is now available. Thanks to everyone who provided feedback on version 0.8.7.

![image](https://user-images.githubusercontent.com/12936/219525409-e1adec37-e012-46f7-8aef-7a0e9926fac2.png)

### Added
 - Schedule plotting on the "Settings | Plotting" page to take advantage of off-peak electricity, lower your fan noise, avoid backups, etc...
 - [Bladebit](https://github.com/guydavis/machinaris/wiki/Bladebit) - alpha GPU plotting support in main Machinaris image. Set `mode: gpuplot` under `bladebit` on Settings | Plotting page.
 - [Gigahorse](https://github.com/guydavis/machinaris/wiki/Gigahorse) - beta GPU plotting and farming support in new Machinaris-Gigahorse image. Set `mode: gpuplot` under `madmax` on Settings | Plotting page.
 - [Ballcoin](https://github.com/ball-network/ballcoin-blockchain) blockchain at version 1.6.0, another slow Silicoin fork.
 - [Pipscoin](https://github.com/Pipscoin-Network/pipscoin-blockchain) blockchain at version 1.7.0.
 - [Chives-Foxy](https://github.com/guydavis/machinaris/wiki/Chives#foxy-pool) - support for the Foxy pool on Chives.  Thanks @fufar!
## Changed
 - Fix missing Connections listing for Flax and MMX blockchains.  Thanks @ekersey!
 - Fix for Bladebit ramplot relaunching.  Thanks @JoeZotacExperience!
 - Multiple functionality & performance cleanups from excellent code review by @qoole.
 - Display compression level for active plotting jobs on Gigahorse and MMX.
## Updated 
 - [BTCGreen](https://github.com/BTCgreen-Network/btcgreen-blockchain/releases/tag/1.7.0b) to v1.7.0b
 - [Cactus](https://github.com/Cactus-Network/cactus-blockchain/releases/tag/1.6.2) to v1.6.2
 - [Chia](https://github.com/Chia-Network/chia-blockchain/releases/tag/1.7.0) to v1.7.0
 - [Chives](https://github.com/HiveProject2021/chives-blockchain/releases/tag/1.5.4) to v1.5.4, including staking.
 - [SHIBGreen](https://github.com/BTCgreen-Network/shibgreen-blockchain/releases/tag/1.7.0.1) to v1.7.0.1
 - [Staicoin](https://github.com/STATION-I/stai-blockchain/releases/tag/1.3.0) to v1.3.0. Note, they require a fresh `config.yaml`.
## Notes
 - Support for new blockchains and tools DOES NOT imply my endorsement for them.  *Only run those you are comfortable with.*

<hr /><em>This discussion was created from the release <a href='https://github.com/guydavis/machinaris/releases/tag/0.8.7'>Version 0.8.7</a>.</em>

**Comment by fufar** [comment]
Some words about chives-foxypool maybe?)

**Reply by guydavis** [reply]
Thanks! Forgot that one.  Been a busy month.  I've added it now.  Thanks for your contribution.

**Reply by fufar** [reply]
I think also better to add it on machinaris.app docker-compose generator or no one will know about it
